packages/terra-worklist-data-grid/src/subcomponents/Row.module.scss

```diff
@@ -7,6 +7,10 @@
     background-color: var(--terra-worklist-data-grid-row-striped-background-color, #f0f0f0);
   }
 
+  .worklist-data-grid-row:nth-child(odd) {
+    background-color: var(--terra-worklist-data-grid-row-striped-background-color, white);
+  }
```

Comment:
Why are we making the background color white?

Reply:
The background color was originally transparent and resulted in overflow columns being visible behind pinned columns. We tested with other themes and this does not seem to impact them.